server/src/domain/system-config/system-config.core.ts

```diff
@@ -164,15 +164,15 @@ export type FeatureFlags = Record<FeatureFlag, boolean>;
 
 let instance: SystemConfigCore | null;
 
-@Injectable()
 export class SystemConfigCore {
   private logger = new ImmichLogger(SystemConfigCore.name);
   private validators: SystemConfigValidator[] = [];
   private configCache: SystemConfigEntity<SystemConfigValue>[] | null = null;
+  private config$ = new Subject<SystemConfig>();
 
-  public config$ = new Subject<SystemConfig>();
-
-  private constructor(private repository: ISystemConfigRepository) {}
+  private constructor(private repository: ISystemConfigRepository) {
+    this.config$.subscribe((newConfig) => repository.emitUpdate(newConfig));
```

Comment:
we can probably delete all of this subscribe stuff.

Reply:
True! Done.